pdf/src/main/groovy/com/craigburke/document/builder/PdfFont.groovy

```diff
@@ -49,11 +49,11 @@ class PdfFont {
 
     static BigDecimal getXHeight(Font font) {
         PDFont pdFont = PdfFont.getFont(font)
-        (font.size * pdFont.getFontHeight(['x'.bytes[0]] as byte[], 0, 1) / 1000f)
+        (font.size * pdFont.getHeight('x'.bytes[0]) / 1000f)
     }
 
     static void addFont(PDDocument document, EmbeddedFont embeddedFont) {
-        PDFont font = PDTrueTypeFont.loadTTF(document, embeddedFont.file)
```

Comment:
unicode handling logic is added in PDType0Font class.
https://pdfbox.apache.org/2.0/migration.html